include/SSVOpenHexagon/Utils/Utils.hpp

```diff
@@ -43,19 +43,39 @@ inline void uppercasify(std::string& s)
     return s;
 }
 
-inline float getFontHeight(sf::Text& font)
+[[nodiscard, gnu::pure]] inline float getFontHeight(sf::Text& font)
 {
     font.setString("A");
     return ssvs::getGlobalHeight(font);
 }
 
-inline float getFontHeight(sf::Text& font, const unsigned int charSize)
+[[nodiscard, gnu::pure]] inline float getFontHeight(sf::Text& font, const unsigned int charSize)
 {
     font.setCharacterSize(charSize);
     font.setString("A");
     return ssvs::getGlobalHeight(font);
 }
 
+bool getLinesIntersection(sf::Vector2f& mIntersection,
+    const sf::Vector2f& l1p1, const sf::Vector2f& l1p2,
+    const sf::Vector2f& l2p1, const sf::Vector2f& l2p2);
+
+unsigned int getLineCircleIntersection(sf::Vector2f& i1,
+    sf::Vector2f& i2, const sf::Vector2f& p1,
+    const sf::Vector2f& p2, const float mRadiusSquared);
+
+bool getLineCircleClosestIntersection(
+    sf::Vector2f& mIntersection, const sf::Vector2f& mPos,
+    const sf::Vector2f& p1, const sf::Vector2f& p2,
+    const float mRadiusSquared);
+
+[[nodiscard, gnu::pure]] inline float fastSqrt(const float &n) 
+{
+    static union{int i; float f;} u;
```

Comment:
```suggestion
    union{int i; float f;} u;
```

Why `static`?

Reply:
I copied-pasted the algorithm from a game development website. Gonna make it static.